customManager/EventManager.ts

```
@@ -0,0 +1,50 @@
+// Event 管理器
+class EventManager {
+  static instance: null | EventManager = null;
+  private eventList: { [key: string]: Function[] } = {};
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)